CHANGELOG.md

```diff
@@ -6,6 +6,53 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- **BREAKING:** Rename `useStaticTokenList` to `useTokenDetection` on the `PreferencesController` constructor options object and set the value to be true by default ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - Token detection will now be enabled by default.
+- **BREAKING:** Append phishfort blocklist to list used by PhishingController ([#715](https://github.com/MetaMask/controllers/pull/715))
+  - The `test` method on `PhishingController` no longer returns a boolean, it now returns an object matching the `EthPhishingDetectResult` interface (defined in `PhishingController.ts`).
+  - Designs may need to be updated to account for the fact that sites may now be blocked by Phishfort. We should ensure users are not directed to the `eth-phishing-detect` repository to dispute Phishfort blocks, because we will not be able to help them.
+- **BREAKING:** Rename `convertPriceToDecimal` method name to `convertHexToDecimal` ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - Consumers of this method must update the function name wherever it is used.
+- Update TokensController to support detected tokens ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - Introduce `detectedTokens` to the config object to track detected tokens, which is updated by calling `addDetectedTokens` whenever TokenDetectionController detects new tokens.
+  - Added `fetchTokenMetadata` private method to fetch token metadata whenever adding individual tokens. This is currently used to populate aggregator information.
+- Append `detectedTokens` to both TokenBalancesController & TokenRatesController `tokens` config object within their `onTokensStateChange` listener methods ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - This change ensures that we are populating both balances and rates for detected tokens.
+- Re-point OpenSea API calls to Codefi proxy ([#805](https://github.com/MetaMask/controllers/pull/805))
+  - All NFT related queries that were routed to OpenSea will now first route to a proxy server owened by Codefi. If this first request fails, and an OpenSea API key has been set, the query will re-route to OpenSea as a fallback.
+- Return ETH currency rate for testnet ETH ([#816](https://github.com/MetaMask/controllers/pull/816))
+- Query for name and symbol on ERC721 contracts regardless of whether or not they support the metadata interface ([#834](https://github.com/MetaMask/controllers/pull/834))
+- Increase polling interval for TokenListController and require minimum of 3 occurences across sources for use in the dynamic token list ([#836](https://github.com/MetaMask/controllers/pull/836))
+
+### Removed
+- **BREAKING:** Deprecate `removeAndIgnoreToken` method in TokensController ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - The logic for removing token(s) is now consolidated in a new method named `ignoredTokens`. Consumers will need to update instances of removeAndIgnoreToken to use ignoredTokens instead.
```

Comment:
Nit: Use backticks around "removeAndIgnoreToken" and "ignoredTokens"?

Reply:
updated here: https://github.com/MetaMask/controllers/pull/848/commits/8f0b66e74d974e5e1da4d250f99b289e2eff70c3